engine/src/externalv1.cpp

```diff
@@ -2094,12 +2205,11 @@ static MCExternalError MCExternalVariableFetch(MCExternalVariableRef var, MCExte
         if (!MCStringConvertToCFStringRef(*t_stringref, *(CFStringRef*)p_value))
             return kMCExternalErrorOutOfMemory;
         
-        // NS types must be autoreleasing
-        if ((p_options & 0xff) == kMCExternalValueOptionAsNSString)
-            [*(NSString**)p_value autorelease];
+        // SN-2014-11-24: [[ Bug 14057 ]] NS types are allocated in an autrelease pool
```

Comment:
Why have these clauses been commented out?

If an NSString is requested, then it must be autoreleased (as we are using CF creation methods)...